src/pages/faq.md

```diff
@@ -23,34 +23,33 @@ import '../customizations/index.css'
 
 <TitleBlock className="custom-text-alignment" slots="heading, text" />
 
-## How can I provide feedback about the API or contact the team with questions?
-
-Please reach out to [psdservices@adobe.com](mailto:psdservices@adobe.com) - we’d be happy to hear from you!
+## How do I purchase and what’s the cost?
 
+We can work with you individually if you’re interested in purchasing the APIs. Reach out to [psdservices@adobe.com](mailto:psdservices@adobe.com) for more information.
```

Comment:
I think this makes sense for enterprise. Do we mention about PAYG somewhere? May be check with Archy/Morgan once?